grammar.js

```diff
@@ -548,13 +542,25 @@ module.exports = grammar({
         ),
       ),
 
-    field_expression: ($) =>
+    field: ($) =>
```

Comment:
I don't see the point of renaming this appart from breaking backward compatibility.